src/coreclr/tools/aot/ILCompiler.ReadyToRun/ObjectWriter/R2RPEBuilder.cs

```diff
@@ -210,16 +161,13 @@ public R2RPEBuilder(
                 _sectionBuilder.SetDllNameForExportDirectoryTable(outputFileSimpleName);
             }
 
-            if (_sectionBuilder.FindSection(R2RPEBuilder.RelocSectionName) == null)
```

Comment:
We create the section builder and add any sections above - there's no need to check if the reloc section already exists.